nlp/2021_11_25_gpt3mix/nlp_gpt3mix.ipynb

```diff
@@ -0,0 +1,1966 @@
+      "metadata": {
+        "id": "4b4GsUwfINep"
+      },
+      "source": [
+        "Now let's print some synthetic samples to examine their quality!"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "6irrOSPrKVU9",
+        "outputId": "27825c0c-ea8f-4738-aec3-b7629b5aa6fc"
+      },
+      "source": [
+        "print(\"Dataset of 10 synthetic samples:\")\n",
+        "print(\"Text: {} \\nLabel: {}\\n\".format(synthetic_gpt3_10_ds['text'][5], idx2label[synthetic_gpt3_10_ds['label'][5]]))\n",
+        "print(\"Dataset of 50 synthetic samples:\")\n",
+        "print(\"Text: {} \\nLabel: {}\\n\".format(synthetic_gpt3_50_ds['text'][5], idx2label[synthetic_gpt3_50_ds['label'][5]]))\n",
+        "print(\"Dataset of 100 synthetic samples:\")\n",
+        "print(\"Text: {} \\nLabel: {}\\n\".format(synthetic_gpt3_100_ds['text'][5], idx2label[synthetic_gpt3_100_ds['label'][5]]))\n",
+        "print(\"Dataset of 200 synthetic samples:\")\n",
+        "print(\"Text: {} \\nLabel: {}\\n\".format(synthetic_gpt3_200_ds['text'][5], idx2label[synthetic_gpt3_200_ds['label'][5]]))"
+      ],
+      "execution_count": null,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "Dataset of 10 synthetic samples:\n",
+            "Text:  even if ur not into these kind of things u have to admit it's pretty cool  \n",
+            "Label: joy\n",
+            "\n",
+            "Dataset of 50 synthetic samples:\n",
+            "Text:  i want to stop running and walk...but the fact that i'm still running is the real miracle  \n",
+            "Label: joy\n",
+            "\n",
+            "Dataset of 100 synthetic samples:\n",
+            "Text:  i want a beer right now  \n",
+            "Label: anger\n",
+            "\n",
+            "Dataset of 200 synthetic samples:\n",
+            "Text:  lol owls  \n",
+            "Label: joy\n",
+            "\n"
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "M6dLc7xMK0NJ"
+      },
+      "source": [
+        "We see that GPT-3 has effectively generated very realistic samples. 👏👏👏"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "K5alF-lekkIr"
+      },
+      "source": [
+        "### GPT-J\n",
+        "\n",
+        "As an open-source alternative of GPT-3, we can use [GPT-J](https://6b.eleuther.ai/) that is a 6 billion parameter model released by a group called Eleuther AI. The goal of the group is to democratize huge LMs, so they released GPT-J and it is currently publicly available. GPT-3 on the other hand, which was released by openAI has 175 billion parameters and is not openly available at the time.\n",
+        "\n",
+        "To use GPT-J, we can simply load it through HuggingFace."
```

Comment:
I would put a note here as well that you need a beefy machine for this, so that it might not be able to load this in a regular Colab Notebook